[2024-05-24T08:54:17.420+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-24T08:54:17.453+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v4.second_task scheduled__2024-05-15T00:00:00+00:00 [queued]>
[2024-05-24T08:54:17.460+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v4.second_task scheduled__2024-05-15T00:00:00+00:00 [queued]>
[2024-05-24T08:54:17.460+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-05-24T08:54:17.481+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): second_task> on 2024-05-15 00:00:00+00:00
[2024-05-24T08:54:17.486+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=2115) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-24T08:54:17.487+0000] {standard_task_runner.py:63} INFO - Started process 2133 to run task
[2024-05-24T08:54:17.489+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v4', 'second_task', 'scheduled__2024-05-15T00:00:00+00:00', '--job-id', '103', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpotjcc31_']
[2024-05-24T08:54:17.497+0000] {standard_task_runner.py:91} INFO - Job 103: Subtask second_task
[2024-05-24T08:54:17.550+0000] {task_command.py:426} INFO - Running <TaskInstance: our_first_dag_v4.second_task scheduled__2024-05-15T00:00:00+00:00 [running]> on host 6bbd6353fbe6
[2024-05-24T08:54:17.682+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gauti' AIRFLOW_CTX_DAG_ID='our_first_dag_v4' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2024-05-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-15T00:00:00+00:00'
[2024-05-24T08:54:17.683+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-24T08:54:17.691+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-24T08:54:17.692+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hai, I am newly created task 2 and I will run after the first task!']
[2024-05-24T08:54:17.703+0000] {subprocess.py:86} INFO - Output:
[2024-05-24T08:54:17.705+0000] {subprocess.py:93} INFO - hai, I am newly created task 2 and I will run after the first task!
[2024-05-24T08:54:17.705+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-24T08:54:17.707+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-24T08:54:17.746+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v4, task_id=second_task, run_id=scheduled__2024-05-15T00:00:00+00:00, execution_date=20240515T000000, start_date=20240524T085417, end_date=20240524T085417
[2024-05-24T08:54:17.795+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-24T08:54:17.823+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-24T08:54:17.827+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
